src/components/shared/search-bar/search-bar.jsx

```diff
@@ -6,7 +6,7 @@ export default function SearchBar({
   setSearchBarView,
   paintingsAndCollections,
 }) {
-  const [searchResults, setSearchResults] = useState(paintingsAndCollections);
+  const [searchResults] = useState(paintingsAndCollections);
```

Comment:
if youre not changing state at any point, dont use it. Just use the original `paintingsAndCollections` variable.